routes/api/v1/foods.js

```diff
@@ -1,8 +1,25 @@
 var express = require("express");
 var router = express.Router();
-// var Food = require('../../../models').Food;
-// var pry = require('pryjs');
-
+var Food = require('../../../models').Food;
+var pry = require('pryjs');
 
+// GET specific food
+router.get('/:id', function(req, res) {
+  Food.findOne({
+    where: {
+      id: req.params.id
+    },
+    attributes: ['id', 'name', 'calories']
+  })
+  .then(food => {
+    if (food !== null) {
```

Comment:
Do we need this to be compared to null?

Reply:
Good call. Don't think we need the `null` after all